Documentation/resources.adoc

```diff
@@ -74,6 +74,43 @@ Expose the user-defined Alertmanager web server within the cluster on the follow
 
 Expose the Alertmanager web server within the cluster on the following ports:
 * Port 9094 provides access to all the Alertmanager endpoints. Granting access requires binding a user to the `monitoring-alertmanager-view` role (for read-only operations) or `monitoring-alertmanager-edit` role in the `openshift-monitoring` project.
+----
+# monitoring-alertmanager-view grants read permissions.
+$ oc project openshift-monitoring
+$ oc create serviceaccount am-ro-client
+$ oc adm policy add-role-to-user monitoring-alertmanager-view \
+  --role-namespace=openshift-monitoring --rolebinding-name=am-ro-client \
+  --serviceaccount=am-ro-client
+$ TOKEN=$(oc create token am-ro-client)
+$ ROUTE=$(oc get route alertmanager-main -n openshift-monitoring -ojsonpath={.spec.host})
+$ curl -H "Authorization: Bearer $TOKEN" -k --fail-with-body "https://$ROUTE/api/v2/alerts?filter=alertname=Watchdog"
+----
+----
+# monitoring-alertmanager-edit grants edit permissions.
+$ oc project openshift-monitoring
+$ oc create serviceaccount am-rw-client
```

Comment:
```suggestion
.List of commands you can run with the `monitoring-alertmanager-edit` role (edit permissions)
[source,terminal]
----
$ oc project openshift-monitoring
$ oc create serviceaccount am-rw-client
```

Reply:
I'll create a ticket for the ascii formatting improvement.

Reply:
opened https://issues.redhat.com/browse/MON-4052